ReactWindows/ReactNative.Shared/Bridge/ReactBridge.cs

```diff
@@ -18,6 +18,7 @@ public class ReactBridge : IReactBridge
         private readonly IJavaScriptExecutor _jsExecutor;
         private readonly IReactCallback _reactCallback;
         private readonly IActionQueue _nativeModulesQueueThread;
+        private bool _batchHadNativeModuleCalls;
 
```

Comment:
Why does this need to be a field on the class, usage seems to be contained to ProcessResponse(JToken, bool)